app/src/main/res/layout/fragment_edit.xml

```diff
@@ -35,19 +38,22 @@
             android:weightSum="2"
             android:layout_marginVertical="8dp">
 
-            <com.google.android.material.textfield.TextInputLayout
+            <app.wefridge.wefridge.EditItemTextInputLayout
                 android:id="@+id/itemQuantityTextInputLayout"
                 android:layout_width="0dp"
                 android:layout_height="wrap_content"
                 android:hint="@string/item_quantity_hint"
-                android:layout_weight="1">
+                android:layout_weight="1"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent">
 
-                <com.google.android.material.textfield.TextInputEditText
+                <app.wefridge.wefridge.EditItemEditText
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
                     android:layout_marginEnd="8dp"
                     android:inputType="number"/>
-            </com.google.android.material.textfield.TextInputLayout>
+            </app.wefridge.wefridge.EditItemTextInputLayout>
 
             <com.google.android.material.textfield.TextInputLayout
```

Comment:
The unit dropdown has a wrong height
![2021-12-13_14-01](https://user-images.githubusercontent.com/22080311/145817148-942f6c88-a6af-4bd7-9a39-b1ec1e455fd7.png)